tests/unit/backend/controllers/admin-forms.server.controller.spec.js

```diff
@@ -241,6 +241,51 @@ describe('Admin-Forms Controller', () => {
     })
   })
 
+  describe('transferOwner', () => {
```

Comment:
Can we move this test to `form.server.model.spec.ts`? And then the integration test in  `admin-forms.server.controller.spec.js` mocks the method and just checks for what is passed to `res` for the success/failure case

Reply:
hmm but in the codebase, most of the schema.methods are being testing in `admin-forms.server.controller.spec.js`